packages/deployer/subtasks/print-info.js

```diff
@@ -25,6 +25,7 @@ async function _printInfo(taskArguments) {
   const network = hre.network.name;
   logger.log(chalk[network.includes('mainnet') ? 'red' : 'gray'](`network: ${network}`));
 
+  logger.log(chalk.gray(`network url: ${hre.network.config.url}`));
```

Comment:
Is important to show the URL for the case the user forgot to remove the fork URL